citizen_report/report/views.py

```diff
@@ -0,0 +1,50 @@
+from django.shortcuts import render, redirect, get_object_or_404
+
+from .models import Report
+from .forms import ReportForm
+
+
+def home(request):
+    return render(request, 'home.html')
+
+
+def show_reports(request):
+    user_reports = Report.objects.filter(user=request.user)
+    current = Report.objects.filter(user=request.user, created_at__isnull=True)
+    return render(request, 'reports.html', {'current': current, 'user_reports': user_reports})
+
+
+def create(request):
+    if request.method == 'GET':
+        return render(request, 'create.html', {'form': ReportForm()})
+    elif request.method == 'POST':
+        form = ReportForm(request.POST)
+        if form.is_valid():
+            report = form.save(commit=False)
+            report.user = request.user
+            report.save()
+            return redirect('reports')
+        else:
+            error = 'something is wrong'
+            return render(request, 'create.html', {'form': ReportForm(), 'error': error})
+    else:
+        error = 'something went wrong'
+        return render(request, 'create.html', {'form': ReportForm(), 'error': error})
+
+
+def detail(request, report_id):
+    report = get_object_or_404(Report, id=report_id, user=request.user)
+    form = ReportForm(instance=report)
+    if request.method == 'GET':
+        return render(request, 'detail.html', {'form': form, 'report': report})
+    elif request.method == 'POST':
+        if form.is_valid():
+            # TODO: user.is_staff
+            form.save()
+            return redirect('tasks')
+        else:
+            error = 'something went wrong'
+            return render(request, 'detail.html', {'form': form, 'report': report, 'error': error})
+    else:
+        error = 'something went wrong'
+        return render(request, 'detail.html', {'form': form, 'report': report, 'error': error})
```

Comment:
po pierwsze, to czy każdy powinien mieć tu wjazd, a dwa, to jeżeli w templatce nie korzystamy z forma to tutaj nie ma sensu obsługiwać posta czy też przekazywać form w kontekście